traffic_ops_ort/t3c-diff-tool/t3c-diff-tool.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/apache/trafficcontrol/traffic_ops_ort/t3cutil"
+	"github.com/kylelemons/godebug/diff"
+	"github.com/pborman/getopt/v2"
+)
+
+func main() {
+	different := false
+	tropsFile := getopt.StringLong("trops-file", 't', "", "Required: Config file name in Traffic Ops")
+	diskFile := getopt.StringLong("disk-file", 'd', "", "Required: Config file on disk")
```

Comment:
If you make these true [Mandatory Options](https://godoc.org/github.com/pborman/getopt/v2#hdr-MANDATORY_OPTIONS), you won't need to check them yourself.